docs/docs/sourcing-from-graphcms.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Sourcing from GraphCMS
+---
+
+## Headless the GraphCMS way
+
+[GraphCMS](https://graphcms.com?ref="gatsby-headless-docs-top") is a headless CMS that is optimized for working with GraphQL. Content structures like posts, authors, products and more are broken down into content types called "models." These models can then be queried with the familiar GraphQL syntax.
+
+One of the benefits of GraphCMS when working with Gatsby is that we support GraphQL natively which allows you to test your content queries before even starting your Gatsby project.
+
+## Getting Started
+
+In this guide we'll walk you through creating a complete project capable of querying data from GraphCMS.
+
+### Install the Boilerplate
+
+To begin, let's create a Gatsby starter site.
+
+```bash
+  # Optionally prefixed with `npx` if you don't have the gatsby cli installed.
+  gatsby new gatsby-site https://github.com/gatsbyjs/gatsby-starter-default
+```
+
+Navigate inside of the project with `cd gatsby-site`.
+
+### Add the Source Plugin
+
+Additionally, you need the the `gatsby-source-graphql` library. Because GraphCMS uses GraphQL natively, we will take advantage of Gatsby's ability to simply stitch two GraphQL APIs together, reducing the time required to transform content. There is no need to use a special gatsby-source-x-cms plugin, the GraphQL source plugin is all you need.
+
+You can install this component with:
+
+```bash
+  # Optionally with `npm install`
+  yarn add gatsby-source-graphql
+```
+
+### Configure the Plugin
+
+The last step required before you can query your data is to configure the `gatsby-source-graphql` plugin. Open `gatsby-config.js` and add the following object to the plugins array. This example uses an open API from GraphCMS but you will most likely want to replace this with your own API and provide a fieldName that makes the most sense to your project.
+
+```js
+{
+    resolve: "gatsby-source-graphql",
+        options: {
+        // The top level query type, can be anything you want!
+        typeName: "GCMS",
+        // The field you'll query against, can also be anything you want.
+        fieldName: "gcms",
+        // Your API endpoint, available from the dashboard and settings window.
+        // You can use this endpoint that features US mountains for now.
+        url: "https://api-euwest.graphcms.com/v1/cjm7tab4c04ro019omujh708u/master",
+    },
+},
+```
+
+If everything works correctly, you should now have your GraphCMS data added to the Gatsby source API!
+
+### Querying for Content
+
+From the root of your project, run the development environment with `yarn run development`. Once the server has started and is error free, you should be able to open the following URL in your browser:
+
+http://localhost:8000/___graphql
+
+This will show you a "Graphiql" playground where you can test your new content API.
```

Comment:
```suggestion
This will show you an interface where you can test your new content API.
```

Playground is a different product, and we're potentially switching from GraphiQL to Playground, so keeping this generic avoids the need to update later.